### PR TITLE
qa_crowbarsetup: use real qa-automation repo

### PR DESCRIPTION
**Pull request by jdsn**
needs fixed https://bugzilla.suse.com/show_bug.cgi?id=927174

we also want to verify the HTTPS cert,
so we use the cert repo+rpm
which is verified by a build key that is not part of SP3,
so we add and verify that first.

### CODE REVIEW

**Comment by jdsn** [comment]
:+1:

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2520,6 +2520,20 @@ function oncontroller()
...
 
+function install_suse_ca()
+{
+    # trust build key - workaround https://bugzilla.opensuse.org/show_bug.cgi?id=935020
+    wget --no-check-certificate -O build.suse.de.key.pgp https://api.suse.de/public/source/SUSE/_pubkey
+    safely sha1sum -c <<EOF
+ee896d59206e451d563fcecef72608546bf10ad6  build.suse.de.key.pgp
+EOF
+    rpm --import build.suse.de.key.pgp
+
```
Comment (MaximilianMeister) [inline scripts/qa_crowbarsetup.sh]: why is this pgp check needed before adding the repo? commit message is a bit unclear to me, and the bug doesn't give an immediate explanation neither.

other than that LGTM

Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: We need to be sure that we are adding the correct repo, so using --auto-import-keys is not an option.

**Comment by MaximilianMeister** [comment]
+1

**Pull request merged**